### ISSUE TITLE
Suggestions to Simplify FQN

### ISSUE THREAD

**Issue by afranioce** [phpactor/phpactor#1258]
Import the class when use full FQN

from
```php
<?php

namespace Foobar\Baz;

class Foo
{
    public function __construct(\Foobar\B<>ar $bar)
    {
    }
}
```

to

```php
<?php

namespace Foobar\Baz;

use Foobar\Bar;

class Foo
{
    public function __construct(Bar $bar)
    {
    }
}
```

**Comment by weeman1337** [comment]
Others™ offer a code action for that

![image](https://user-images.githubusercontent.com/6216686/121145656-77014980-c83f-11eb-9ece-5c989a02b530.png)

This action can be applied to FQNs. It replaces all occurrences of the FQN in the current document by a short name with an import.

**Comment by dantleech** [comment]
"classic" Phpactor also has an `:ExpandClass` action in VIM, so it would make sense to add both add code actions.

**Comment by mamazu** [comment]
Would that be the same as ExpandClass but only the other way around? Then I can try to give it a shot.

**Comment by dantleech** [comment]
Yes would be possible. Note that expand class doesn't have a code action ATM, maybe it could be a toggle.

**Comment by mamazu** [comment]
So basically `:ExpandClass` and `:ExpandClass true` would turn the class name to an FQN and `:ExpandClass false` would turn the FQN into just the class name with an import?

**Comment by dantleech** [comment]
I would just implement it as a code action.
